### PR TITLE
Update dark-mode styling of map controls

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

Make the map control buttons and selectors look nicer and more similar to the toggles.
Also updated styling on a bunch of tooltips that were missed earlier.

### Preview

Screenshot 2023-09-04 at 00 18 59

### CODE REVIEW

File: web/src/features/charts/PriceChart.tsx [hunk 1]
```diff
@@ -44,11 +44,11 @@ function PriceChart({ datetimes, timeAverage }: PriceChartProps) {
...
       <ChartTitle translationKey="country-history.electricityprices" />
-      <div className="relative">
+      <div className="relative overflow-hidden">
         {isPriceDisabled && (
-          <div className="absolute top-0 h-full w-full">
-            <div className=" h-full w-full rounded bg-white opacity-50 dark:bg-gray-700" />
-            <div className="absolute left-[50%] top-[50%] z-10 w-60 -translate-x-1/2 -translate-y-1/2 rounded-sm bg-gray-200 p-2 text-center text-sm shadow-lg dark:bg-gray-900">
+          <div className="absolute top-0 -ml-3 h-full w-[115%]">
+            <div className="h-full w-full rounded bg-white opacity-90 dark:bg-gray-900" />
+            <div className="absolute left-[45%] top-[50%] z-10 w-60 -translate-x-1/2 -translate-y-1/2 rounded-sm bg-gray-200 p-2 text-center text-sm shadow-lg dark:border dark:border-gray-700 dark:bg-gray-800">
```
Comment (madsnedergaard) [inline web/src/features/charts/PriceChart.tsx:51]: **Before:**
Screenshot 2023-09-04 at 00 24 56

**After:**
Screenshot 2023-09-04 at 00 27 19

**Review by VIKTORVAV99 — APPROVED** [review]
The changes in here looks very good IMO. Nice to see that we are unifying and modernizing it a bit!

Just saw something small though that I thought we could include in this pr, the spacing around the + and - buttons are not consistant and I would prefer if we updated the top spacing to match the bottom one:
Skärmavbild 2023-09-04 kl  09 18 17

**Comment by madsnedergaard** [comment]
Nice catch, fixed :)

**Comment by madsnedergaard** [comment]
cc @rasmusjeppsen

**Review by VIKTORVAV99 — COMMENTED** [review]
Noticed one more thing when blinding myself with light mode. The buttons are not transparent like the toggles. Do we want to change that in this PR?

**Pull request merged**